src/target/source/codegen_source_base.cc

```diff
@@ -28,42 +28,22 @@ namespace tvm {
 namespace codegen {
 
 void CodeGenSourceBase::ClearFuncState() {
-  name_alloc_map_.clear();
+  name_supply_->Clear();
```

Comment:
what if we just constructed a new NameSupply here instead? the potential for abuse in a pass seems high

Reply:
Good point. I removed the method.